rbac/core/kafka.py

```diff
@@ -34,16 +38,33 @@ class RBACProducer:
 
     def get_producer(self):
         """Init method to return fake kafka when flag is set to false."""
+        retries = 0
+        max_retries = settings.KAFKA_AUTH.get("retries")
         if not hasattr(self, "producer"):
             if settings.DEVELOPMENT or settings.MOCK_KAFKA or not settings.KAFKA_ENABLED:
                 self.producer = FakeKafkaProducer()
+                logger.info("Fake Kafka producer initialized in development mode")
             else:
-                if settings.KAFKA_AUTH:
-                    self.producer = KafkaProducer(**settings.KAFKA_AUTH)
-                elif not settings.KAFKA_SERVERS:
-                    raise AttributeError("Empty servers list")
-                else:
-                    self.producer = KafkaProducer(bootstrap_servers=settings.KAFKA_SERVERS)
+                while retries <= max_retries:
+                    try:
+                        if settings.KAFKA_AUTH:
+                            self.producer = KafkaProducer(**settings.KAFKA_AUTH)
```

Comment:
You have to return KafkaProducer(**settings.KAFKA_AUTH) if it succeeds. Or it would retry even if it succeeds because you put it under while loop. Can we add a unit test for this?